src/electron/electron/handler.cljs

```diff
@@ -105,9 +102,9 @@
                                                                 " failed, "
                                                                 e
                                                                 (when backup-path
-                                                                  " A backup file was saved to "
-                                                                  backup-path
-                                                                  "."))}))))))
+                                                                  (str ". A backup file was saved to "
```

Comment:
Minor bug fix - The backup file message wasn't visible because only the last value of the `when` block is returned - "."